Formula/o/opentap.rb

```diff
@@ -0,0 +1,83 @@
+class Opentap < Formula
+  desc "Fast and easy development and execution of automated tests"
+  homepage "https://opentap.io"
+  url "https://github.com/opentap/opentap.git",
+    tag:      "v9.24.2",
+    revision: "bafe5eb48b0bcad2bd0f07198b69b6a58608807b"
+  license "MPL-2.0"
+
+  depends_on "dotnet"
+
+  uses_from_macos "unzip" => :build
+
+  def install
+    full_version = "9.24.0+2569356e"
```

Comment:
that is not good

Reply:
No, I agree. Is there another way of getting the git sha of the tag?